ramls/tags.raml

```diff
@@ -0,0 +1,58 @@
+#%RAML 1.0
+title: mod-kb-ebsco-java
+baseUri: https://github.com/folio-org/mod-kb-ebsco-java
+protocols: [ HTTPS ]
+version: v1
+mediaType: "application/vnd.api+json"
+
+documentation:
+  - title: mod-kb-ebsco-java
+    content: Implements the eholdings interface using EBSCO KB as backend.
+
+traits:
+  pageable: !include traits/pageable.raml
+
+types:
+  tagCollection: !include types/tags/tagCollection.json
+  jsonapiError: !include types/jsonapiError.json
+  recordType:
+    type: string
+    description: Eholdings record type. Possible values are provider, package, title, resource.
+  recordTypes:
+    type: recordType[]
+
+/eholdings/tags:
+  displayName: Tags
+  get:
+    description: Retrieve a collection of tags associated with eholding resords.
+    queryParameters:
+      filter[rectype]?:
```

Comment:
Is the ? at the end intentional?

Reply:
means optional property
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#general

Reply:
👍